main.py

```diff
@@ -503,7 +504,15 @@ def main(cl_arguments):
     if args.do_target_task_training:
         # Train on target tasks
         pre_target_train_path = setup_target_task_training(args, target_tasks, model, strict)
-
+        # Check for previous target train checkpoints
+        task_directory = check_for_previous_checkpoints(
+            args.run_dir, target_tasks, "target_train", args.load_model
+        )
+        if len(task_directory) > 0:
```

Comment:
For checkpoints elsewhere, we return None rather than "". Let's use that convention.